packages/snap/src/utils/generate-snapshots.ts

```diff
@@ -4,52 +4,31 @@ import webpack from 'webpack';
 import React from 'react';
 import {HTMLSnapshotPlugin} from '@stylable/webpack-extensions';
 import {renderToStaticMarkup} from 'react-dom/server';
-import {getCompName, IComponentMetadata, IRegistry} from '@ui-autotools/registry';
-import {consoleLog, consoleWarn} from '@ui-autotools/utils';
-import {dedent} from './dedent';
-import {parseSnapshotFilename} from './filename-utils';
-import { IFileInfo } from './build-base-files';
+import {consoleWarn} from '@ui-autotools/utils';
+import { createHtml } from './create-html';
+import { ISimSnapInfo } from '../types';
 
-function findComponentByName(name: string, Registry: IRegistry): IComponentMetadata<any, any> | void {
-    // We only have to do this because we currently map the component definitions to their metadata,
-    // not the names. So we have no way to get component metadata by name. And at this point in the build
-    // process, the component definition returned by webpack has been modified from the original, so we can't
-    // get the metadata with it
-  for (const component of Registry.metadata.components) {
-    const metadata = component[1];
-    if (getCompName(metadata.component) === name) {
-      return metadata;
-    }
-  }
+export interface IFileInfo {
+  basename: string;
+  filepath: string;
 }
 
-export function render(fileName: string, Registry: IRegistry, compiledFile: any, sourceFile: any) {
-  const compMetadata = findComponentByName(compiledFile.default.name, Registry);
+export function render(fileName: string, snapInfo: ISimSnapInfo, returnHtml: (html: string) => void, compiledFile: any, _sourceFile: any) {
```

Comment:
If this is only related to stylable move it to the stylable folder. Why does `_sourceFile` have an underline?

Reply:
It's unused but we need a param after it. The _ makes the linter happy